src/app/shared/date.service.ts

```diff
@@ -70,7 +69,14 @@ export class DateService {
         return time + (time === 1 ? ' day ago' : ' days ago');
       }
     } else {
-      return 'n/a';
+      return null;
+    }
+  }
+  getISO8601Format(timestamp: Date | number): string | null {
```

Comment:
You're doing this because the dbCreateDate is actually a number, even though according to the TypeScript model, it should be a Date, right?

It seems a little hacky, but I think the real problem is that dbCreateDate is not the type that it is declared to be (it's a number, but according to src/app/shared/swagger/model/entry.ts, it should be a Date).

@garyluu Do you have any insight into this? Is this a Swagger codegen issue?

Reply:
The swagger.yaml shows:
```
        type: "string"
        format: "date-time"
```
so the UI Swagger codegen seems correct in thinking it's a Date.  The webservice, on the other hand, is returning a number.  So this is ether some serialization issue or that the swagger.yaml is not generated correctly.

Reply:
Should I make a separate ticket for that? is this solution ok for the time being?

Reply:
I think a separate ticket is fine. Please reference this code in the ticket so whoever tackles the ticket will also remember to come back and update this when/if they fix the underlying issue.

Reply:
https://ucsc-cgl.atlassian.net/browse/DOCK-832